meta-zephyr-sdk/recipes-devtools/arc_qemu/files/cross.patch

```diff
@@ -0,0 +1,30 @@
+We need to be able to trigger configure's cross code but we don't want
+to set cross_prefix as it does other things we don't want. Patch things
+so we can do what we need in the target config case.
+
+Upstream-Status: Inappropriate [may be rewritten in a way upstream may accept?]
+Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
+
+
+Index: qemu/configure
+===================================================================
+--- a/configure
++++ b/configure
+@@ -3934,7 +3934,6 @@ if test "$skip_meson" = no; then
```

Comment:
Offsets differ from the same patch for generic QEMU, so maybe you copy over one version over another so that they match exactly? Otherwise it might look a bit suspicious.